specs/sifive.com/PRCI/RESET/0.1.0/RESET_rtl.json5

```diff
@@ -0,0 +1,42 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'PRCI',
+    name: 'RESET_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'PRCI',
+      name: 'RESET',
+      version: '0.1.0',
+    },
+    ports: {
+      RESET: {
+        description: 'reset signals',
+        wire: {
+          isReset: true,
+          onMaster: {direction: 'out', presence: 'required'},
+          onSlave:  {direction: 'in', presence: 'required'}
+        }
+      }
+    },
+    props: {
+      ResetPolarity: {
+          type: "string",
+	  enum: ["ActiveHigh", "ActiveLow"],
+      },
+      ResetType: {
+	  type: "string",
+	  enum: ["Synchronous", "Asynchronous", "Full Asynchronous"]
+      },
+      ClocksBeforeDeassertion: {
+	  type: "integer",
```

Comment:
should we assume `minimum: 0` here?

Reply:
Megan is probably right that 0 for Async* and 1 for Sync. for the before deassertion case.

Reply:
I added min of 0